### ISSUE TITLE
Funny bug (or feature?): page zoom level also applies to popup menu

### ISSUE THREAD

**Issue by byzod** [Robbendebiene/Gesturefy#694]
Popup menu from gesturefy and vanilla context menu, 100% zoom
> ![1](https://github.com/Robbendebiene/Gesturefy/assets/848607/72e98163-6814-49a9-b5c2-3601fa70e4e1)

---

Vanilla context menu and popup menu from gesturefy, 400% zoom 🤣
> ![2](https://github.com/Robbendebiene/Gesturefy/assets/848607/7650dd52-45dc-441a-8c35-2611e9e04475)

**Comment by Robbendebiene** [comment]
The explanation is that the entire Gesturefy UI (like the gesture trail) scales, simply because it hast to be part of the website.
I would have prefered using the native context menu for the popup gestures if it was possible.
Since it is not I'm left with two options:
1. mimic the context menu as good as possible (e.g. counter scale the popup when the user zooms in)
2. make a virtue out of necessity - treat the popup as something own in other words independent of the context menu GUI element

I prefer the second option for several reasons:
- mimicking will always fall short which is displeasing when encountered (cannot use the OS style, will not cross browser window and many more)
- not strictly following the context menu pattern allows me to choose the best bang for the bug when it comes to implementing good user interaction patterns that play nicely with mouse gestures (in fact I already diverged in some smaller points)

So the interesting question is, does scaling the popup cause any harm?
I don't have a strong opinion on that but at least from an accessibility point of view scaling the menu is definitely the way to go as it allows visual impaired people to increase the readability of the menu.

Long story short - For me this is a _wontfix_ until somebody comes up with a good reason to counter scale the menu that is worth the effort and also does not defeat the accessibility aspect.
